src/hid/hid-device.cpp

```diff
@@ -273,6 +313,12 @@ namespace librealsense
             if(fps > 0)
                 featureReport.report = (1000 / fps);
 
+            //we want to change the sensitivity values only in gyro, for FW version>=5.16
+            if( (int)featureReport.reportId == REPORT_ID_GYROMETER_3D
```

Comment:
Please change all the defines like `REPORT_ID_GYROMETER_3D` to consts or enums under librealsense::platform namespace.
Then no casting wil l be necessary.

Reply:
Done